var/spack/repos/builtin/packages/subversion/package.py

```diff
@@ -67,9 +67,9 @@ def install(self, spec, prefix):
         options.append('--with-sqlite=%s' % spec['sqlite'].prefix)
         options.append('--with-serf=%s' % spec['serf'].prefix)
 
-        if spec.satisfies('^swig'):
+        if 'swig' in spec:
             options.append('--with-swig=%s' % spec['swig'].prefix)
-        if spec.satisfies('+perl'):
```

Comment:
It did seem to work for me, but perl was being pulled into the spec by openssl, so I wasn't sure if it works in general. I replaced it as a precaution, but I can put it back if you prefer.